person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @rentals = []
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
+  def of_age?
```

Comment:
- Optional. 
Since the previous reviewer didn't raise this as required, I am marking this as optional.
Method `of_age?` should be a private method as per the project requirements. Kindly make this method private.